src/views/DefineCompany.vue

```diff
@@ -9,7 +9,7 @@
 
     <section class="mt-10">
       <header>
-        <h2>1. Company Name</h2>
+        <h2>1. Name</h2>
```

Comment:
I still find this design change odd, as it will end up saying `Name` twice for numbered company IA's but alas!

![Screen Shot 2021-08-16 at 10 26 17 AM](https://user-images.githubusercontent.com/53542131/129604654-5f84120b-747b-4585-8bca-08f56de62ae8.png)

Reply:
No action required obviously, i know you are just following design here :)

Reply:
I think Tracey is aware of this and decided it was OK.
(Nice catch.)

Reply:
kk sounds good!

Reply:
Background: some entities are not "companies".

Reply:
Good point!